fix(button): submit button when in form (#21)
ovh/design-system: packages/contributing/contributing-deployment-and-versioning.mdx

### Minor: add extra features

You added one or several features **without any breaking changes** in the branch `release/next`.

> Create a release branch from up-to-date `release/next` that corresponds to the version you are going to release like `release/X.Y`.

Reviewer: ```suggestion
Create a release branch from up-to-date `master` that corresponds to the version you are going to release like `release/X.Y`.
```

Then, you have to generate a minor version from your branch.

```bash
yarn ods:minor
```

The release branch will be used for bugfixes later (patch).
It will be used for bug fixes later (patch).
Once the support for the version has ended, the branch will be un-referenced.

### Major: breaking changes

> You added breaking changes in the `release/next` branch:

Reviewer: ```suggestion
You added breaking changes in the `master` branch:
```

> Create a release branch from up-to-date `release/next` that corresponds to the version you are going to release like `release/X.Y`.

Reviewer: ```suggestion
Create a release branch from up-to-date `master` that corresponds to the version you are going to release like `release/X.Y`.
```
